tests/js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"scripts": {
+		"test": "npm run test:js:watch",
+		"test:js": "cross-env BABEL_ENV=test NODE_ENV=test jest --config=./jest.config.js --passWithNoTests",
+		"test:js:watch": "npm run test:js -- --watch"
+	},
+	"dependencies": {
+		"@googlesitekit/assets": "file:../../assets"
+	},
+	"devDependencies": {
+		"@jackfranklin/test-data-bot": "^2.0.0",
+		"@shopify/jest-dom-mocks": "^4.0.2",
+		"@testing-library/jest-dom": "^5.16.5",
+		"@testing-library/react": "^10.4.9",
+		"@testing-library/react-hooks": "3.3.0",
+		"@wordpress/babel-preset-default": "^4.17.0",
+		"@wordpress/jest-preset-default": "7.1.1",
+		"cross-env": "^7.0.3",
+		"element-internals-polyfill": "^1.3.5",
+		"faker": "^5.5.3",
+		"fetch-mock": "^9.10.3",
+		"fetch-mock-jest": "^1.3.0",
+		"jest": "^26.6.3",
+		"jest-localstorage-mock": "^2.4.2",
+		"jsdom": "^16.5.0",
```

Comment:
Why do we need it here? Where is it used?

Reply:
@eugene-manuilov This is used for storybook test workflow.